bin/run-ansible.sh

```diff
@@ -421,24 +421,32 @@ else
     echo ""
 fi
 
-ansible_executable=~/.local/bin/ansible-playbook
-if [[ ! -f "${ansible_executable}" ]]; then
+# Awful hack to make bionic checks pass (they need to be disabled)
+if [[ $codename == *"bionic"* ]]; then
+    ansible_executable=~/.local/bin/ansible-playbook
+    if [[ ! -f "${ansible_executable}" ]]; then
+        ansible_executable=ansible-playbook
+    fi
+    ansible_basic_executable=~/.local/bin/ansible
+    if [[ ! -f "${ansible_basic_executable}" ]]; then
+        ansible_basic_executable=ansible
+    fi
+    ansible_galaxy_executable=~/.local/bin/ansible-galaxy
+    if [[ ! -f "${ansible_galaxy_executable}" ]]; then
+        ansible_galaxy_executable=ansible-galaxy
+    fi
+ else
     ansible_executable=ansible-playbook
-fi
-ansible_basic_executable=~/.local/bin/ansible
-if [[ ! -f "${ansible_basic_executable}" ]]; then
     ansible_basic_executable=ansible
-fi
-ansible_galaxy_executable=~/.local/bin/ansible-galaxy
-if [[ ! -f "${ansible_galaxy_executable}" ]]; then
     ansible_galaxy_executable=ansible-galaxy
+    # Use conda packages/modules first
+    export PYTHONPATH="${shadow_conda_ws_dir}/lib/python${aurora_python_version}/site-packages"
```

Comment:
`aurora_python_version` doesn't seem to be defined in this script

Reply:
it's a little bit gross, but 
https://github.com/shadow-robot/aurora/blob/0194890571c11d74aade0cb612eb5ab26b825da4/bin/conda_utils.sh#L25 -> https://github.com/shadow-robot/aurora/blob/0194890571c11d74aade0cb612eb5ab26b825da4/bin/run-ansible.sh#L360

Reply:
Oh, I see.